src/main/java/org/allenai/wikitables/WikiTablesDataProcessor.java

```diff
@@ -0,0 +1,67 @@
+package org.allenai.wikitables;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import edu.stanford.nlp.sempre.Formula;
+import edu.stanford.nlp.sempre.FuzzyMatchFn.FuzzyMatchFnMode;
+import edu.stanford.nlp.sempre.tables.TableKnowledgeGraph;
+import edu.stanford.nlp.sempre.tables.test.*;
+import edu.stanford.nlp.sempre.tables.match.*;
+import fig.basic.*;
+
+public class WikiTablesDataProcessor {
+  private ArrayList<Pair<String, String>> pairedPaths;
+
+  public WikiTablesDataProcessor(Map<String, String> dataPaths) {
+    // dataPaths contains a map of datatype strings to their paths.
+    // Eg: {"train": "/path/to/train", "dev": "/path/to/dev"}
+    pairedPaths = new ArrayList<Pair<String, String>>();
+    Iterator it = dataPaths.entrySet().iterator();
```

Comment:
I think it's cleaner to replace this iterator and while loop with a for loop like this:

for (Map.Entry<String, String> entry : dataPaths.entrySet()) { 
...
}